jina/parsers/__init__.py

```diff
@@ -17,6 +17,7 @@ def set_pod_parser(parser=None):
 
     from jina.parsers.orchestrate.base import mixin_scalable_deployment_parser
     from jina.parsers.orchestrate.pod import mixin_pod_parser, mixin_hub_pull_options_parser
+    from hubble.executor.parsers.pull import mixin_hub_pull_options_parser
```

Comment:
I did on purpose to have Jina's own parser to reuse install requirements so that I can edit the description